sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtilTest.java

```diff
@@ -772,7 +773,11 @@ public void testGCSChannelCloseIdempotent() throws IOException {
         GoogleCloudStorageReadOptions.builder().setFastFailOnNotFound(false).build();
     SeekableByteChannel channel =
         new GoogleCloudStorageReadChannel(
```

Comment:
nit : We could remove the dependency on `GoogleCloudStorageReadChannel` and use the higher level of abstraction of `GCSImpl`

Reply:
good point. done.